client/settings/payment-settings/general-settings-section.js

```diff
@@ -30,32 +29,7 @@ const GeneralSettingsSection = () => {
 					) }
 				/>
 
-				<CheckboxControl
```

Comment:
Moved this to a separate component with the confirmation modal, to encapsulate a few thins